python_modules/libraries/dagster-azure/dagster_azure/adls2/io_manager.py

```diff
@@ -132,3 +132,57 @@ def my_job():
         init_context.resource_config.get("adls2_prefix"),
     )
     return pickled_io_manager
+
+
+class PickledObjectADLS2AssetIOManager(PickledObjectADLS2IOManager):
+    def _get_path(self, context):
+        return "/".join([self.prefix, *context.asset_key.path])
```

Comment:
i assume this is how all the other asset io managers work?

Reply:
yep!